### ISSUE TITLE
Support PyTorch1.0

### ISSUE THREAD

**Issue by kamo-naoyuki** [espnet/espnet#514]
Please add to the milestones of v0.3.0 #444 :

- Support PyTorch1.0: https://github.com/pytorch/pytorch/releases/tag/v1.0.0
- Remove deprecated warning from PyTorch

```
torch/nn/functional.py:52: UserWarning: size_average and reduce args will be deprecated, please use reduction='elementwise_mean' instead.
```

I haven't tested pytorch1.0 at all.

**Comment by kamo-naoyuki** [comment]
Note: I won't help about this matter.

**Comment by sw005320** [comment]
Thanks for your suggestion.
I added them to our milestone #444.

**Comment by ShigekiKarita** [comment]
Or maybe we can use pytorch's own torch.nn.functional.ctc_loss instead. https://pytorch.org/docs/stable/nn.html#ctc-loss

**Comment by kamo-naoyuki** [comment]
Completed at #553 
I also removed deprecated warning from PyTorch.
